manifests/charts/istio-control/istio-discovery/templates/validatingwebhookconfiguration.yaml

```diff
@@ -4,6 +4,7 @@ kind: ValidatingWebhookConfiguration
 metadata:
   name: istio-validator{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}-{{ .Values.global.istioNamespace }}
   labels:
+    # FIXME: Should this have the owning resource label?
```

Comment:
It probably doesn't matter except for back compat - the operator is going away and we would use more normative labels like https://github.com/istio/istio/blob/master/manifests/charts/ztunnel/templates/_helpers.tpl anyway.